modules/Sources/Features/NHSendFlow/NHScan/NHScanView.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  NHScanView.swift
+//  
+//
+//  Created by Matthew Watt on 7/22/23.
+//
+
+import ComposableArchitecture
+import Generated
+import SwiftUI
+import UIComponents
+import Utils
+
+public struct NHScanView: View {
+    let store: NHScanStore
+    
+    public init(store: NHScanStore) {
+        self.store = store
+    }
+    
+    public var body: some View {
+        WithViewStore(store) { viewStore in
+            GeometryReader { geometry in
+                VStack {
+                    NighthawkHeading(
+                        title: L10n.Nighthawk.TransferTab.Scan.scanPaymentRequest,
+                        subtitle: L10n.Nighthawk.TransferTab.Scan.scanPaymentRequestDetails
+                    )
+                    .padding(.bottom, 40)
+                    
+                    QRCodeScanView(
+                        rectOfInterest: normalizedRectOfInterest(geometry.size),
+                        onQRScanningDidFail: { viewStore.send(.scanFailed) },
+                        onQRScanningSucceededWithCode: { viewStore.send(.scan($0.redacted)) }
+                    )
+                    .frame(
+                        width: geometry.size.width * 0.7,
+                        height: geometry.size.width * 0.7
+                    )
+                    .cornerRadius(8)
+                    .overlay(
+                        CornerStrokeSquare()
+                            .stroke(Asset.Colors.Nighthawk.peach.color, lineWidth: 5)
+                            .cornerRadius(2)
+                            .padding(18)
+                    )
+                    
+                    Spacer()
+                }
+                .showNighthawkBackButton(action: { viewStore.send(.backButtonTapped) })
+            }
+            .onAppear { viewStore.send(.onAppear) }
+            .onDisappear { viewStore.send(.onDisappear) }
+        }
+        .applyNighthawkBackground()
+    }
+}
+
+// MARK: - Subviews
+private extension NHScanView {
+    func frameSize(_ size: CGSize) -> CGFloat {
+        size.width * 0.55
+    }
+
+    func rectOfInterest(_ size: CGSize) -> CGRect {
+        CGRect(
+            x: size.width * 0.5,
+            y: size.height * 0.5,
+            width: frameSize(size),
+            height: frameSize(size)
+        )
+    }
+
+    func normalizedRectOfInterest(_ size: CGSize) -> CGRect {
+        CGRect(
```

Comment:
[blocking] 

parameter size is not used. Consider using a computed var or even a let if this is correct.